src/theming/themes/fruit-os-dark.yaml

```diff
@@ -0,0 +1,54 @@
+display-name: fruitOS Dark
+variant: dark
+# Basics
+accent: "#e54a5c"
+accent-secondary: "#F25769"
+background: "#27282aa8"
+overlay-bg: "#252527ec"
+# Album
+album-playing-shadow: var(--accent)
+album-playing-play-bg: "#2f2c2c"
+album-playing-play-hover-bg: var(--accent)
+album-playing-title-bg: var(--accent)
+# Analytics/Stats
+# Artist's Toolkit
+# Library
+library-text: "#ffffff"
+library-odd-row-bg: "#363b3a7e"
+library-playing-bg: "#d12f47"
+library-hover-bg: "#45454571"
+library-playing-text: "#ffffff"
+library-highlight-bg: "#d0536655"
+library-header-bg: "#2a2f2ebf"
+library-header-active-bg: "#4b514fab"
+library-header-text: "#acb4ac"
+library-playing-icon: "#ffffff"
+library-scrollbar-bg: "#827f8a7e"
+library-scrollbar-hover-bg: "#b3afbb77"
+library-favourite-icon: "#e54a5c"
+library-favourite-hover-icon: "#e54a5c"
+# Map View
+# Menus
+# Oscilloscope
+oscilloscope: "#e54a5c"
+# Panels
+panel-background: "#2b302f77"
+panel-primary-border-main: rgba(255, 255, 255, 0.2)
+panel-secondary-border-main: rgba(255, 255, 255, 0.05)
+# Popups
+# Progress bar
+# Prune
```

Comment:
How does the template work? So it's ok to omit certain values, but the section comments must remain? Does the order matter?

Reply:
When loading, the `core` themes (not `base16`, `dark` or `light`) are merged with their `root` theme (`dark` or `light`).
So the order the values or comments don't matter.
It's just for consistency and ease of management.

Reply:
Got it, thanks!